### PR TITLE
feat: add drush spec

### PR DESCRIPTION
**Pull request by andeersg**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
Missing support for drush autocomplete.

**What is the new behavior (if this is a feature change)?**
Autocomplete support for drush.

**Additional info:**

- [x] Basic autocomplete for commands

### CODE REVIEW

File: src/drush.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  description: string;
+  default: null | string | Array<string>;
+}
+
+interface DrushOption {
+  name: string;
+  shortcut: string;
+  accept_value: boolean;
+  is_value_required: boolean;
+  is_multiple: boolean; // not supported by fig
```
Comment (fedeci) [inline src/drush.ts:14]: Is this what you are looking for? https://fig.io/docs/reference/option#isrepeatable

Comment (andeersg) [reply src/drush.ts:14]: It's a property from drush output, but I guess there is no need to have it defined here since it's not used. Will remove them 👍

Comment (andeersg) [reply src/drush.ts:14]: That's true, will fix it 👍

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**